hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueContainer.java

```diff
@@ -980,24 +986,29 @@ public boolean hasEnoughCapacity(int delta) {
      * @return backup replica map from item ID to queue item
      */
     public Map<Long, QueueItem> getBackupMap() {
+        ConcurrentMap<Long, QueueItem> backupMap = this.backupMap;
         if (backupMap == null) {
+            Queue<QueueItem> itemQueue = this.itemQueue;
```

Comment:
This mix of referring to a local variable and the field is a bit confusing. Can you elaborate a bit or simply use the local variable only when checking the size and adding to backup map?

Reply:
now it should be more clear